web/profiles/joinup/config/install/user.role.authenticated.yml

```diff
@@ -7,8 +7,7 @@ weight: 1
 is_admin: false
 permissions:
   - 'access content'
-  - 'create collection rdf entity'
```

Comment:
Thanks! Very important. I've fixed it and added tests to ensure that nobody can access the form to create a collection any more.